docs/integrations/payment-gateways.md

```diff
@@ -0,0 +1,76 @@
+---
+id: payment-gateways
+title: Supported Payment Gateways
+---
+
+Saleor supports the following payment gateways:
+
+- `DUMMY` - for testing purposes only!
+- `BRAINTREE` - supports PayPal and Credit Cards
+- `RAZORPAY` - supports only the paisa currency
+- `STRIPE` - supports Credit Cards
+
+The default configuration only uses the dummy backend. It is meant to allow developers to easily simulate different payment results.
+
+> **Note**
+>
+> All payment backends default to using sandbox mode. This is very useful for development. When deploying to a production server, make sure you use the production mode.
+
+### Braintree 
+
+This gateway implements payments using [Braintree](https://www.braintreepayments.com/).
+
+| Environment Variable | Description |
```

Comment:
We're no longer using environment variables, should refer to configuration options in plugin instead.